fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBDatabaseFactory.java

```diff
@@ -366,6 +378,26 @@ public synchronized void setRunLoopProfilingEnabled(boolean runLoopProfilingEnab
         this.runLoopProfilingEnabled = runLoopProfilingEnabled;
     }
 
+    /**
+     * Set the number of threads per FDB client version. The default value is 1.
+     *
+     * @param tPerClientV the number of threads per client version. Cannot be less than 1
+     */
+    public static synchronized void setThreadsPerClientVersion(int tPerClientV) {
```

Comment:
minor: I think it might be a little clearer if the parameter was `threadsPerClientVersion` instead of `tPerClientV`

Reply:
changed to `threadsPerClientV`, which is the best I could do with a static field named `threadsPerClientVersion`, sadly. But more clear, at least

Reply:
Why not shadow the variable and then set it in the method with `FDBDatabaseFactory.threadsPerClientVersion = threadsPerClientVersion`?